pkgs/applications/emulators/openmsx/default.nix

```diff
@@ -73,3 +73,4 @@ stdenv.mkDerivation rec {
     platforms = platforms.unix;
   };
 }
+# TODO: updater script
```

Comment:
I'm not sure if this should be here. Guess it doesn't really matter in the grand scheme of things, but not sure.

Reply:
The idea is to help r-ryantm, because it doesn't know how to convert the `x_y` to `x.y` automatically.

Reply:
I get that part (and would happily review such an addition) -- I'm talking about merging this file with this comment.

Reply:
Indeed, I will remove it.
I need to reorganize my backlogs another way.